frontend/src/components/calendar/event-dialog/EventDialog.js

```diff
@@ -278,6 +187,8 @@ function EventDialogContent(props) {
           <p>{event.description}</p>
 
           <section>
+            {userSignUpJSX}
+
             <h6>Pending ({pending?.length || 0})</h6>
             {pending?.map((volunteer, index) => {
               return (
```

Comment:
Prevent Signed up volunteers from seeing this info

Reply:
This is only viewable for admins.

Reply:
This is what I meant, when I click "Sign Up" I am put directly in Approved category and I can see everyone else who is also signed up for the event. Not sure if we want this functionality or not
<img width="582" alt="Screen Shot 2021-07-18 at 5 45 21 AM" src="https://user-images.githubusercontent.com/41653027/126067604-291fc8f8-96d5-43bc-bf78-a1ee2785cfe0.png">

Reply:
I changed it so it only shows those approved in the latest commit. For approval discussion, see above.

Reply:
Awesome, thank you Ben!!! PR is good to merge, Fin emailed our rep about a meeting today.